File: get_token.py
import paramiko

client = paramiko.SSHClient()
client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
client.connect('187.127.164.142', username='root', password='Truedial@1111')

cmd = """docker exec fmi_backend php artisan tinker --execute="
\$user = \\App\\Models\\User::where('email', 'test12345@test.com')->first();
\$token = \$user->createToken('admin')->plainTextToken;
echo 'TOKEN: ' . \$token;
" """

stdin, stdout, stderr = client.exec_command(cmd)
print("STDOUT:", stdout.read().decode())
print("STDERR:", stderr.read().decode())

File: scripts/get_token.php
<?php
require '/var/www/html/vendor/autoload.php';
$app = require '/var/www/html/bootstrap/app.php';
$kernel = $app->make(Illuminate\Contracts\Http\Kernel::class);
$app->boot();
$user = App\Models\User::where('role', 'admin')->orWhere('email', 'like', '%admin%')->first()
    ?? App\Models\User::first();
if ($user) {
    $token = $user->createToken('diag-token')->plainTextToken;
    echo "USER: " . $user->email . " | ROLE: " . $user->role . "\n";
    echo "TOKEN: " . $token . "\n";
} else {
    echo "No users found\n";
}
